app/Views/errors/html/error_exception.php

```diff
@@ -5,32 +5,32 @@
 	<meta charset="UTF-8">
 	<meta name="robots" content="noindex">
 
-	<title><?= esc($title) ?></title>
+	<title><?= esc($title); ?></title>
```

Comment:
I usually don't use a semicolon on echoes, it doesn't make sense to me to have them and it makes scanning PHP in HTML easier to distinguish. Is this configurable or is it a bundle deal? Is this a PSR-12 requirement or an addition?

Reply:
This isn't in PSR12. And is not configurable. I'll just disable this as consensus wants otherwise.